src/raft_log.rs

```diff
@@ -184,11 +214,12 @@ impl<T: Storage> RaftLog<T> {
             if !self.match_term(e.index, e.term) {
                 if e.index <= self.last_index() {
                     info!(
-                        "{} found conflict at index {}, [existing term:{}, conflicting term:{}]",
-                        self.tag,
-                        e.index,
-                        self.term(e.index).unwrap_or(0),
-                        e.term
+                        self.logger,
+                        "{tag} found conflict at index {index}, [existing term: {existing_term}, conflicting term: {conflicting_term}]",
```

Comment:
Seems `existing_term` and `conflicting_term` could be part of the structured data here.